README.md

```diff
@@ -79,6 +103,7 @@ The URL to add is `http://localhost:3000/webhook`. Don't forget to substitue the
 
 Supported WebHook Events currently are:
 - Application Evaluation
+- Violation Alert
```

Comment:
This is a new Webhook, but no additional TYPE has been added?

Reply:
You are right, violation alert actually has a different header. Will remove this for now!